File: test_2.py
from Controller import SaveGame
import requests

s = SaveGame(20260320)

# json = {"points_earned":1,"words_played":["tide"],"total_points":"237"}
# s.write_save(json)

# game = 20260515
# requests.post (f"127.0.0.1:5000/save/{game}")

# print (s.check_save())
print (s.load_save())

File: test_2.js
async function sendData (endpoint_url, data) {
    const response = await fetch(endpoint_url, {
        method: "POST",
        body: JSON.stringify(data),
        headers: {
            "Content-Type": "application/json"
        },
    });
    const text = await response.text();
    return text;
}
// const data = {"points_earned":1,"words_played":["tide"],"total_points":"237"}
// await postData("127.0.0.1:5000/save/" + 20260515, data)